### DISCUSSION TITLE
Momentary "Fire" or "Ping" upon value change (Feature request)

### DISCUSSION THREAD

**Discussion by CaptainBobSim** [MobiFlight/MobiFlight-Connector#765] [Ideas]
**Is your feature request related to a problem? Please describe.**
It would be super awesome to have an event that triggers for a defined amount of time then turns back off.

For me this is useful for circuit breakers that need to pop out for 3 seconds then stay off to prevent overheating.
Ping using buzzer for cockpit warnings that only sound when entering a strange situation

**Describe the solution you'd like**
A "seconds to stay on for" dropdown under display for an LED/Output.

**Describe alternatives you've considered**
This can be coded through RPN, but that doesn't allow for use with simulators other than FS2020

**Additional context**
This request is likely not helpful to 99% of the community, but might be cool to see for uses in solenoids/electromagnets.

**Comment by DocMoebiuz** [comment]
This is already possible to some extent. You can use the new Display Input Actions to trigger an Event and have it write the current Sim Time plus your timeout to a variable.
Then you can make a light go on as long as the current Sim Time is smaller than the value at the MobiFlight Variable.

**Comment by DocMoebiuz** [comment]
workaround available. Something like this will be possible once I overhaul the transform engine.
